packages/contracts-bedrock/test/L1/OPContractsManager.t.sol

```diff
@@ -236,11 +238,12 @@ contract OPContractsManager_Upgrade_Harness is CommonTest {
 
 contract OPContractsManager_Upgrade_Test is OPContractsManager_Upgrade_Harness {
     function runUpgradeTestAndChecks(address delegateCaller) public {
+        vm.etch(delegateCaller, vm.getDeployedCode("test/mocks/Callers.sol:DelegateCaller"));
+
```

Comment:
just thought this was something better done upfront rather than after assertions.